lib/keen/styles/variables.scss

```diff
@@ -33,9 +33,8 @@ $ui-input-height: rem(32px) !default;
 $ui-input-margin-bottom: rem(16px) !default;
 
 // Input label
-$ui-input-label-color: $secondary-text-color !default;
```

Comment:
Even though it has been removed, there are still a couple of places in the file `lib/KSelect/index.vue` where `ui-input-label-color` is used, this could be causing some of the tests to fail.

Reply:
usages of `$ui-input-label-color` still exist in KSelect